ingest/expression_files/mtx.py

```diff
@@ -202,6 +206,7 @@ def get_features(feature_row: str):
         return gene_id, gene_name
 
     @staticmethod
+    @custom_metric(config.get_metric_properties, "sortPerfTime", {"sorted": False})
```

Comment:
Please use `perfTime:sort` (generally `<perfTime:<foo>`), [like `single_cell_portal_core`](https://github.com/broadinstitute/single_cell_portal_core/pull/716):

```suggestion
    @custom_metric(config.get_metric_properties, "perfTime:sort", {"sorted": False})
```

Reply:
317fded